xbmc/media/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(SOURCES MediaType.cpp)
 
-set(HEADERS MediaType.h)
+set(HEADERS MediaLock.h
+            MediaType.h)
```

Comment:
Minor request, I've started doing

```c++
set(HEADERS MediaLock.h
            MediaType.h
)
```

to improve the Git history when lexicographically trailing files are added